### PR TITLE
Python_module into develop

### PR DESCRIPTION
**Pull request by Setnea**

### CODE REVIEW

File: python/modules/task.py [hunk 1]
```diff
@@ -451,6 +452,22 @@ def execute_specify(self, env_local, dict_setts):
...
+                 "python_script_path" : {"required" : True}}
+
+    def __init__(self, parameters, updates,  args = {}):
+        TASK.__init__(self, parameters, updates, args)
+
+    def execute_specify(self, env_local, dict_setts):
+        external_params = list(self.dict_task.keys())
+        logger.info("Executing python function %s from %s", dict_setts["python_function_name"], dict_setts["python_script_path"]) 
+        param_dict = python_connection.prepare_param_dict(env_local, self.parameters, external_params)
+        output_dict = python_connection.execute_python_script(param_dict, dict_setts["python_script_path"], dict_setts["python_function_name"])
```
Comment (stork119) [inline python/modules/task.py:467]: Lines are too long, add some enters

File: python/modules/python_connection.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/python
+import importlib.util
+import logging
+
+logger = logging.getLogger("IPIQA.python_connection")
+
+def execute_python_script(param_dict, python_script_path, function_name):
+    p_source = importlib.util.spec_from_file_location("external_module", python_script_path)
```
Comment (stork119) [inline python/modules/python_connection.py]: Add some comment this function. How script should look like ? Do you load python script and then execute functions that are there ? 
What if python script is bigger than one file ?

**Review by stork119 — CHANGES_REQUESTED** [review]
@Setnea  try to answer on my questions

File: python/modules/task.py (hunk 1 above)
Comment (Setnea) [reply python/modules/task.py:467]: Fixed, where possible.

**Review by stork119 — APPROVED** [review]

File: python/modules/python_connection.py (hunk 2 above)
Comment (Setnea) [reply python/modules/python_connection.py]: Added some comments which explains how to use module. 
Yes, I load it as module, get an assigment for function with defined name and than execute it.
If scripts are bigger than one file, user should take care of proper imports i.e. import other modules by using their absolute paths.

**Pull request merged**